### DISCUSSION TITLE
Displaying the DME distance

### DISCUSSION THREAD

**Discussion by drose030** [nguyenquyhy/Flight-Tracker-StreamDeck#189] [Q&A]
Hi, 

is there currently a way to display the DME distance? I couldn’t find the right variable.
Simconnect seems to have it as `NAV DME:index`

David

**Comment by nzeppos** [comment]
I think the latest commit https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/commit/e35b759bc1cb9e9d43031bdfab199bcb9fe708b1 might be what you're looking for?

So I assume DME is coming.

**Reply by nguyenquyhy** [reply]
I actually added those variables due to this request, but I can't find the request again in issue list. Now I understand why 😂.

**Reply by drose030** [reply]
Very awesome, thanks 🙏

**Reply by nguyenquyhy** [reply]
I have released this in version 0.29.

By the way, next time please use issue list (instead of Discussion) for feature request. It is much easy for me to track :).

**Reply by drose030** [reply]
Will do, thanks 😊
